internal/resource/http/http.go

```diff
@@ -176,10 +176,18 @@ func (h *resourceHandler) handleDelete(w http.ResponseWriter, r *http.Request, c
 
 func parseParams(r *http.Request) (tenancy *pbresource.Tenancy, params map[string]string) {
 	query := r.URL.Query()
+	namespace := query.Get("namespace")
+	if namespace == "" {
+		namespace = query.Get("ns")
+	}
+	peer := query.Get("peer")
+	if peer == "" {
+		peer = query.Get("peer_name")
+	}
```

Comment:
Have we agreed to changing `peer_name` to `peer`?

Reply:
The HTTP API expects the query param to be `peer_name` which is consistent with the gRPC but the CLI [calls it](https://github.com/hashicorp/consul/blob/main/api/api.go#L125) `peer`. So we either support both or update CLI to add these [here](https://github.com/hashicorp/consul/blob/main/api/api.go#L834).

I am inclined to support both rather over an inconsistent experience within CLI. WDYT?

Reply:
I agree on supporting both to keep retro compatibility but also provide a way of having a consistent UX. Maybe we should also deprecate peer in preparation of removing it? 🤔